src/panel/category/CategoryPanel.jsx

```diff
@@ -134,10 +137,28 @@ const CategoryPanel = ({ poiFilters = {}, bbox }) => {
     </>;
   }
 
+  const NavHeader = () => {
+    if (isMobile) {return null;}
```

Comment:
I wonder if it's worth creating another component for that…
At least you can simplify it by moving the last `return null` into this early return test.